src/libraries/System.Linq/ref/System.Linq.cs

```diff
@@ -36,7 +36,7 @@ public static partial class Enumerable
         public static double? Average<TSource>(this System.Collections.Generic.IEnumerable<TSource> source, System.Func<TSource, long?> selector) { throw null; }
         public static float? Average<TSource>(this System.Collections.Generic.IEnumerable<TSource> source, System.Func<TSource, float?> selector) { throw null; }
         public static float Average<TSource>(this System.Collections.Generic.IEnumerable<TSource> source, System.Func<TSource, float> selector) { throw null; }
-        public static System.Collections.Generic.IEnumerable<TResult> Cast<TResult>(this System.Collections.IEnumerable source) { throw null; }
+        public static System.Collections.Generic.IEnumerable<TResult?> Cast<TResult>(this System.Collections.IEnumerable source) { throw null; }
```

Comment:
If memory serves, there are a few places in System.Data.Common that will need !s.

Reply:
fixed